content/zh/_index.md

```diff
@@ -54,7 +53,7 @@ OpenTelemetry 是各类 API、SDK 和工具形成的集合。可用于仪表化
 
 {{% blocks/feature icon="fas fa-chart-line" title="链路、指标、日志" %}}
 
-从你的服务和软件中创建并采集遥测数据，然后将其转发给各种分析工具。
+从你的服务和软件中生成并采集遥测数据，然后将其转发给各种分析工具。
```

Comment:
"创建" -> "生成"

The previous translation for the word "create" is correct. But it might be more properate to use the word "generate" (in Chinese) here. 

In Chinese, we will typically say "generate ... based on ..." rather than "create ... based on ..." (It can be "create ... from ..." but I think it's not match the context here, since here we mean generating telemetry data based on the action / operation of your application.

Please have me feedback on this if possible.